snap/info.go

```diff
@@ -323,6 +327,32 @@ func (s *Info) Services() []*AppInfo {
 	return svcs
 }
 
+func (s *Info) BadInterfacesInfoString() string {
```

Comment:
This would be nicer going into a function of its own. It seems too specialized and too custom-formatted to make sense in Info itself.

The function might look like this, at the package level:
```
func BadInterfacesSummary(snapInfo *snap.Info) string
```
I think we'll also want to change the formatting soon so it's more readable, but we can do that in a follow up.